### ISSUE TITLE
How did you save the binary weights in your code?

### ISSUE THREAD

**Issue by Shuang1997** [TianweiXing/BNN#1]
Hi, I want to modify the resnet-50 to a BNN version in tensorflow.  I fellow the BNN's paper that I have a set of weights in float "w" and also a set of  binary weights "bin_w", using the gradients of "bin_w" to update "w", but I don't know how to transfer the float weight to binary after training and save them in my checkpoint file, can you help me?

**Comment by TianweiXing** [comment]
Just use the binarization function you defined to binarize the weights. I 'm not sure if TF supports saving binary weights in checkpoint file now, but at least you can save the weights as np array and load it next time you use.
